APITestingFramework/StepDefinitions/User/Get.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Text.Json;
+using Core;
+using Features.GeneralSteps;
+using Models;
+using RestSharp;
+using TechTalk.SpecFlow;
+
+namespace Features.User.Get
+{
+    [Binding]
+    [Scope(Feature = "Retrieve an existing user")]
+    public class GetStepDefinitions : CommonSteps
```

Comment:
the class name should be the same that the class

Reply:
Done